### PR TITLE
[SPARK-20749][SQL][FOLLOW-UP] Override prettyName for bit_length and octet_length

### PR DESCRIPTION
**Pull request by gatorsmile**
## What changes were proposed in this pull request?
We need to override the prettyName for bit_length and octet_length for getting the expected auto-generated alias name.

## How was this patch tested?
The existing tests

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @kiszk @cloud-fan

File: sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4 [hunk 1]
```diff
@@ -141,7 +141,7 @@ statement
         (LIKE? pattern=STRING)?                                        #showTables
     | SHOW TABLE EXTENDED ((FROM | IN) db=identifier)?
         LIKE pattern=STRING partitionSpec?                             #showTable
-    | SHOW DATABASES (LIKE? pattern=STRING)?                            #showDatabases
+    | SHOW DATABASES (LIKE? pattern=STRING)?                           #showDatabases
```
Comment (HyukjinKwon) [inline sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4]: Nit: seems unrelated?

**Review by HyukjinKwon — APPROVED** [review]
LGTM

**Comment by cloud-fan** [comment]
LGTM

**Comment by HyukjinKwon** [comment]
Merged to master and branch-2.3.
